oqsprov/oqsprov.c

```diff
@@ -226,87 +363,93 @@ static const OSSL_ALGORITHM oqsprovider_signatures[] = {
     SIGALG("rsa3072_sphincsshake128fsimple", 128, oqs_signature_functions),
 #endif
-///// OQS_TEMPLATE_FRAGMENT_SIG_FUNCTIONS_END
-    { NULL, NULL, NULL }
-};
+    ///// OQS_TEMPLATE_FRAGMENT_SIG_FUNCTIONS_END
+    {NULL, NULL, NULL}};
 
 static const OSSL_ALGORITHM oqsprovider_asym_kems[] = {
 ///// OQS_TEMPLATE_FRAGMENT_KEM_FUNCTIONS_START
 #ifdef OQS_ENABLE_KEM_frodokem_640_aes
-    KEMBASEALG(frodo640aes, 128)
-    KEMHYBALG(p256_frodo640aes, 128)
-    KEMHYBALG(x25519_frodo640aes, 128)
+    KEMBASEALG(frodo640aes, 128) KEMHYBALG(p256_frodo640aes, 128) KEMHYBALG(
+        x25519_frodo640aes, 128)
 #endif
 #ifdef OQS_ENABLE_KEM_frodokem_640_shake
-    KEMBASEALG(frodo640shake, 128)
-    KEMHYBALG(p256_frodo640shake, 128)
-    KEMHYBALG(x25519_frodo640shake, 128)
+        KEMBASEALG(frodo640shake, 128) KEMHYBALG(
+            p256_frodo640shake, 128) KEMHYBALG(x25519_frodo640shake, 128)
 #endif
 #ifdef OQS_ENABLE_KEM_frodokem_976_aes
-    KEMBASEALG(frodo976aes, 192)
-    KEMHYBALG(p384_frodo976aes, 192)
-    KEMHYBALG(x448_frodo976aes, 192)
+            KEMBASEALG(frodo976aes, 192) KEMHYBALG(
+                p384_frodo976aes, 192) KEMHYBALG(x448_frodo976aes, 192)
 #endif
 #ifdef OQS_ENABLE_KEM_frodokem_976_shake
-    KEMBASEALG(frodo976shake, 192)
-    KEMHYBALG(p384_frodo976shake, 192)
-    KEMHYBALG(x448_frodo976shake, 192)
+                KEMBASEALG(frodo976shake, 192) KEMHYBALG(
+                    p384_frodo976shake, 192) KEMHYBALG(x448_frodo976shake, 192)
 #endif
 #ifdef OQS_ENABLE_KEM_frodokem_1344_aes
-    KEMBASEALG(frodo1344aes, 256)
-    KEMHYBALG(p521_frodo1344aes, 256)
+                    KEMBASEALG(frodo1344aes, 256) KEMHYBALG(p521_frodo1344aes,
+                                                            256)
 #endif
 #ifdef OQS_ENABLE_KEM_frodokem_1344_shake
-    KEMBASEALG(frodo1344shake, 256)
-    KEMHYBALG(p521_frodo1344shake, 256)
+                        KEMBASEALG(frodo1344shake,
+                                   256) KEMHYBALG(p521_frodo1344shake, 256)
 #endif
 #ifdef OQS_ENABLE_KEM_kyber_512
-    KEMBASEALG(kyber512, 128)
-    KEMHYBALG(p256_kyber512, 128)
-    KEMHYBALG(x25519_kyber512, 128)
+                            KEMBASEALG(kyber512, 128) KEMHYBALG(
+                                p256_kyber512, 128) KEMHYBALG(x25519_kyber512,
+                                                              128)
 #endif
 #ifdef OQS_ENABLE_KEM_kyber_768
-    KEMBASEALG(kyber768, 192)
-    KEMHYBALG(p384_kyber768, 192)
-    KEMHYBALG(x448_kyber768, 192)
-    KEMHYBALG(x25519_kyber768, 128)
-    KEMHYBALG(p256_kyber768, 128)
+                                KEMBASEALG(kyber768, 192) KEMHYBALG(
+                                    p384_kyber768,
+                                    192) KEMHYBALG(x448_kyber768,
+                                                   192) KEMHYBALG(x25519_kyber768,
+                                                                  128) KEMHYBALG(p256_kyber768,
```

Comment:
Now, this formatting really looks ugly. Shouldn't the stuff get more readable?

Reply:
Agreed. Thanks! But some other places didn't improve likewise (?) See latest comment...